go/arrow/internal/arrjson/arrjson_test.go

```diff
@@ -3101,4 +3102,640 @@ func makeDurationsWantJSONs() string {
 
 func makeDecimal128sWantJSONs() string {
 	return `` // FIXME(fredgan): implement full decimal128 JSON support
-}
\ No newline at end of file
+}
+
+func makeMapsWantJSONs() string {
+	return `{
```

Comment:
for testing purposes would it make sense to use a shorter example?

Reply:
because these tests are generated from the `arrdata` record batches that's why this ends up this large. The reason why those records are the size they are is to ensure that we're properly testing handling multiple chunks with a map and multiple records.